live-examples/css-examples/fonts/meta.json

```diff
@@ -63,6 +63,15 @@
             "title": "CSS Demo: font-style",
             "type": "css"
         },
+        "fontStretch": {
+            "baseTmpl": "tmpl/live-css-tmpl.html",
+            "cssExampleSrc":
+                "../../live-examples/css-examples/fonts/font-stretch.css",

```

Comment:
This example isn't working for me, I think because this path is wrong. Please see https://github.com/mdn/interactive-examples/blob/master/CONTRIBUTING-CSS.md#updating-the-metadata.